.github/workflows/pr-check.yml

```
@@ -0,0 +1,13 @@
+name: PR Check
+on: pull_request
+jobs:
+  gradle:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - uses: eskatos/gradle-command-action@v1
+      with:
+        arguments: check 
```

```suggestion
        arguments: check --no-daemon
        wrapper-cache-enabled: true
        dependencies-cache-enabled: true
        configuration-cache-enabled: true
```

The third cache line doesn't bring much, but doesn't hurt either.